File: cfgs/pythoncfgs/MAXIMUM_SUM_2_X_N_GRID_NO_TWO_ELEMENTS_ADJACENT.py
Nodes
1;statement;foo_1 = fun_1(foo_2[0][0], foo_3[1][0]);8;
2;statement;foo_1 = 0;9;
3;conditional;foo < foo;10;
4;statement;return fun_1(foo_1, foo_2);14;
5;statement;foo_1 = fun_1(foo_2, foo_3);11;
6;statement;foo_1 = foo_2 + fun_1(foo_3[0][foo_4], foo_5[1][foo_6]);12;
7;statement;foo_1 = foo_2;13;
8;exit;;
Edges
0;2;
1;3;
2;4;5;
3;8;
4;6;
5;7;
6;3;
7;

File: cfgs/java_cfgs_foo/MAXIMUM_SUM_2_X_N_GRID_NO_TWO_ELEMENTS_ADJACENT.java
Nodes
1;special;;;
2;statement;int incl = fun_1(foo_1[0][0], foo_2[1][0]);14;
3;statement;int excl = 0;15;
4;statement;int excl_new;15;
5;statement;int i = 1;16;
6;conditional;foo_1 < foo_2;17;
7;statement;excl_new = fun_1(foo_1, foo_2);19;
8;statement;incl = foo_1 + fun_1(foo_2[0][foo_3], foo_4[1][foo_5]);20;
9;statement;excl = foo_1;21;
10;statement;i++;18;
11;statement;return fun_1(foo_1, foo_2);23;
12;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;
5;7;11;
6;8;
7;9;
8;10;
9;6;
10;12;
11;
